etc/helm/pachyderm/values.yaml

```diff
@@ -208,11 +208,18 @@ loki-stack:
       enabled: true
       accessModes:
         - ReadWriteOnce
-      size: 5Gi
+      size: 10Gi
       storageClassName: standard
       annotations: {}
       nodeSelector: {}
       tolerations: []
+    config:
+      limits_config:
+        retention_period: 24h
+        retention_stream:
+          - selector: '{suite="pachd"}'
+            priority: 1
+            period: 168h # = 1 day
```

Comment:
I assume you mean 1 week. It looks like YAML supports `1d` and `1w`, which might be preferable

Reply:
> It looks like YAML supports 1d and 1w, which might be preferable

The [Loki docs](https://grafana.com/docs/loki/latest/configuration/#generic-placeholders) indicate that durations must match this regex: `[0-9]+(ns|us|µs|ms|[smh])`; units can only be nanoseconds, microseconds, milliseconds, seconds minutes and hours.

You are completely correct about 1 week — my fault. Fixed in latest.